docs/administering/for-work.md

```diff
@@ -0,0 +1,112 @@
+[Sandstorm for Work](https://sandstorm.io/business) is the name for a collection of features that
+are typically used by companies. Customers can buy a _feature key_ to enable these features on a
+Sandstorm server running at their own organization. Special pricing, sometimes including free
+feature keys, are available to [non-profits and community groups](#special-pricing). For pricing,
+please read the information on the [main website](https://sandstorm.io/business). As an
+implementation detail, Sandstorm is fully open-source software.
+
+For now, feature keys are typically 60-day trial keys. This gives you time to evaluate Sandstorm for
```

Comment:
The one I issued to square is 90-day, though I don't know that I'm informed enough to have an opinion.